pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -38,7 +38,8 @@ type AccountSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	AwsAccountID  string `json:"awsAccountID"`
+	AwsAccountID string `json:"awsAccountID"`
+	// +optional
```

Comment:
Why was this added?

Reply:
forgot to remove it after removing `AwsAccountOU`